tegra-fuse.c

```diff
@@ -173,26 +183,36 @@ tegra_fuse_context_open (const char *basepath)
 	unsigned long chipid;
 	char soctype[65];
 
-	fd = open("/sys/module/tegra_fuse/parameters/tegra_chip_id", O_RDONLY);
-	if (fd < 0)
-		return NULL;
+	fd = open("/sys/module/tegra_fuse/parameters/tegra_chip_uid", O_RDONLY);
```

Comment:
It's just `tegra_chip_id` for the old sysfs entry, not `tegra_chip_uid`, right?

Reply:
We support both in the old `sysfs` 
It makes the things easy to manage below.
I tried this changes with both `Jetson xavier nx` which have `Jetpack 4.6.3`/`L4T R32.7.3` and `Jetson Orin AGX` whcih have `Jetpack 5.1` / `L4T R35.2.1`

Reply:
OK, I see. It would have helped if you had mentioned that somewhere.

I've looked, and that newer `chip_uid` parameter wasn't added until L4T R32.5.  I would prefer sticking with the older `chip_id` parameter and retaining compatibility with earlier versions, if you don't mind.

Reply:
Sorry about not mentioning that somewhere. I checked from `L4T R32.6.x` which aligned with our active branches.
I will apply the necessary changes as you preference.
No worries